docs/README.md

```diff
@@ -1,68 +1,89 @@
+
+**Source Code**: <a href="https://github.com/Mirascope/mirascope" target="_blank">https://github.com/Mirascope/mirascope</a>
 
 ---
 
 ## Why use Mirascope?
 
-- **Easy**: Designed for ease of use, Mirascope features an intuitive interface and a minimal learning curve
-- **Intuitive**: Get editor support for prompts, eliminating the need to dig through documentation
-- **Durable**: Mirascope offers versatile and adaptable functionality, allowing seamless integration with custom solutions
-- **Integration**: Leveraging Pydantic, Mirascope offers easy integration with JSON Schema and other tools
-- **Maintainability**: Reduce prompt-related bugs and organize prompts with provided utilities
+**Mirascope** is a library purpose-built for Prompt Engineering on top of <a href="https://pydantic.dev" target="_blank">Pydantic</a>:
 
-## Installation
+* **Intuitive**: Editor support that you expect (e.g. **autocompletion**, **inline errors**)
+* **Peace of Mind**: Pydantic together with our Prompt CLI **eliminate prompt-related bugs**.
+* **Durable**: Seamlessly customize and extend functionality. **Never maintain a fork**.
+* **Integration**: Easily integrate with **JSON Schema** and other tools such as **FastAPI**
+* **Convenience**: Tooling that is **clean**, **elegant**, and **delightful** that **you don't need to maintain**.
+* **Open**: Dedication to building **open-source tools** you can use with **your choice of LLM**.
 
-Install Mirascope and start building with LLMs in minutes.
+## 🚨 Warning: Strong Opinion
 
-```sh
-$ pip install mirascope
-```
+Prompt Engineering is just engineering. Beyond simple toy examples (that aren't useful), prompting quickly becomes complex. Separating prompts from the engineering workflow will only put limitations on what you can build with LLMs. We firmly believe that prompts are far more than "just f-strings" and thus require developer tools that are purpose-built to make building these more complex prompts as easy as possible.
 
-This will install the `mirascope` package and CLI.
+Furthermore, the best developer experience requires the following, which we strive to uphold:
 
-## A Simple Mirascope Example
+* *Simplicity through idiomatic syntax* -> ***Faster and more reliable releases***
+* *Semi-opinionated methods* -> ***Reduced complexity that speeds up development***
+* *Reliability through validation* -> ***More robust applications with fewer bugs***
+* *High-quality up-to-date documentation* -> ***Trust and reliability***
 
-```python
-from mirascope import OpenAIChat, Prompt
+Speeds up development
+Results in fewer bugs and iterations
+Helps to guarantee robust applications
+Enables faster and more reliable releases
+Reduces complexity and risk through maximal composability and customizability
 
-class BookRecommendationPrompt(Prompt):
-    """
-    Can you recommend some books on {topic}?
-    """
+Once you go mirascope, you'll never go back...ascope
```

Comment:
...

Reply:
so you like this right?

Reply:
The phrase this is based on makes this problematic.

Reply:
i think something cheesy is fine, but lets brainstorm better ones

Reply:
I'm happy to just remove, this was mostly a joke cause I couldn't think of something actually good